test/specs/Conformance-test.js

```diff
@@ -14,37 +12,20 @@ describe('Conformance', () => {
   /* eslint-disable no-console */
   console.info('Conformance-test renders each component with no props, required prop warnings may occur.')
   /* eslint-enable no-console */
-  _.each(stardust, (SDComponent, name) => {
-    const classes = faker.fake('{{hacker.noun}} {{hacker.noun}} {{hacker.noun}}')
-    const sdClass = getSDClassName(name)
-    const firstChild = _.first(render(<SDComponent />).children())
+  _.each(componentInfo, (info) => {
+    const { Component, constructorName, filenameWithoutExt, sdClass } = info
 
-    describe(name, () => {
+    describe(constructorName, () => {
       it('extends Component', () => {
-        expect(SDComponent.prototype).to.eql(Component.prototype)
-      })
-
-      it(`constructor name is "${name}" (matches component name)`, () => {
-        SDComponent.prototype.constructor.name.should.equal(name)
+        expect(Component.prototype).to.eql(React.Component.prototype)
```

Comment:
I appreciate the clarity of `equal` over `eql`. Looks like we may be able to avoid API look-ups if we use the `deep` in the chain and stick with `equal`.